### PR TITLE
feat(UI-1013): adjust login page to mobile devices

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Adjust login page to mobile devices

## Linear Ticket
https://linear.app/autokitteh/issue/UI-1013/responsive-login-page

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/pages/login.tsx
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:1]: done

File: src/components/pages/login.tsx [hunk 1]
```diff
@@ -27,12 +27,12 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
 	const benefitsList = Object.values(t("rightSide.benefitsList", { returnObjects: true }));
 
 	return (
-		<div className="flex h-screen">
-			<div className="relative flex w-2/3 items-center justify-center rounded-3xl pb-32 pl-16 text-black">
-				<div className="z-10 p-5">
+		<div className="flex flex-col-reverse sm:h-screen sm:flex-row">
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:30]: done

File: src/components/pages/login.tsx [hunk 2]
```diff
@@ -27,12 +27,12 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
...
+			<div className="relative flex w-full items-center justify-center p-4 pb-32 text-black sm:w-2/3 sm:rounded-3xl sm:pl-16">
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:31]: done

File: src/components/pages/login.tsx [hunk 3]
```diff
@@ -27,12 +27,12 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
...
+				<div className="z-10 flex flex-col p-2 sm:p-5">
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:32]: done

File: src/components/pages/login.tsx [hunk 4]
```diff
@@ -27,12 +27,12 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
...
 					<AHref
 						ariaLabel={t("leftSide.logoText")}
-						className="absolute left-4 top-4 flex h-auto items-center"
+						className="relative flex h-auto items-center sm:absolute sm:left-4 sm:top-4"
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:35]: done

File: src/components/pages/login.tsx [hunk 5]
```diff
@@ -61,26 +61,24 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
 				</div>
 				<LazyLoginLogos />
 			</div>
-			<div className="z-10 flex w-7/12 shrink-0 flex-col items-center justify-center rounded-l-2xl bg-gray-1250 p-8 font-averta text-white">
+			<div className="z-10 flex w-full shrink-0 flex-col items-center justify-center bg-gray-1250 p-8 py-12 font-averta text-white sm:w-7/12 sm:rounded-l-2xl">
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:64]: done

File: src/components/pages/login.tsx [hunk 6]
```diff
@@ -61,26 +61,24 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
...
 				<h1 className="text-center font-averta text-4xl font-semibold">
-					{t("leftSide.welcomeTitle")}
-					<br />
-					{t("leftSide.autokittehGreenTitle")}
+					{t("leftSide.welcomeTitle")} {t("leftSide.autokittehGreenTitle")}
 				</h1>
+				<h2 className="mt-6 text-xl">Log in or Sign up</h2>
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:68]: done

File: src/components/pages/login.tsx [hunk 7]
```diff
@@ -61,26 +61,24 @@ const Login = ({ descopeRenderKey, handleSuccess, isLoggingIn }: LoginPageProps)
...
-					{t("leftSide.signupText")}{" "}
 					<AHref
-						className="font-averta text-green-800 hover:text-gray-500"
+						className="font-averta text-green-800 underline hover:text-gray-500"
 						href="https://autokitteh.com/get-a-demo/"
 						relationship="noreferrer"
 						target="_blank"
 					>
-						{t("leftSide.signupLink")}
+						Website
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/pages/login.tsx:81]: done

**Review by J1za — APPROVED** [review]

**Pull request merged**